### ISSUE TITLE
Use Babel JSX transform to obviate need to import React

### ISSUE THREAD

**Issue by jhwohlgemuth** [FormidableLabs/spectacle#1082]
### Description

I love Spectacle.  I have used myriad presentation frameworks and Spectacle is one of my favorites.

That being said, there are certain tasks I find myself performing after using `spectacle-boilerplate`...like adding the new JSX transform.

With the release of React v17, it is possible to [use Babel to  auto import React](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html).  I recommend that `spectacle-boilerplate` leverage this functionality.  React v17 and Babel (`preset-react`) are already being used, so there is basically no additional overhead introduced to newly created presentations.

This allows the developer to remove `import React from 'react';` from all their modules.  A small, but not insignificant improvement IMO.

> **NOTE**: This only *really* matters when like me, you tend to break your slides into separate files and have to include `import React from 'react';` at the top of each file...even when `React` is not used explicitly.

### Proposal

I recommend that we update [the boilerplate webpack config file](https://github.com/FormidableLabs/spectacle-cli/blob/main/config/webpack.config.js#L14) to read:
```diff
- presets: [require.resolve('@babel/preset-react')]
+ presets: [[require.resolve('@babel/preset-react'), {runtime: 'automatic'}]]
```
I am perhaps missing some consequence of this change, but it has not bitten me yet...

> **NOTE**: I am willing to make a PR, if there is sufficient interest.

### Links / References

- [Introducing the New JSX Transform](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html)

**Comment by carlos-kelly** [comment]
Great suggestion. We will have this rolled into our `9.0.0` release.

**Comment by carlos-kelly** [comment]
Shipped in `v9.0.0`
